wies/core/views.py

```diff
@@ -290,107 +455,26 @@ def get_context_data(self, **kwargs):
         else:
             context['next_page_url'] = None
 
-        return context
-
-
-class AssignmentDetailView(DetailView):
-    model = Assignment
-    template_name = 'assignment_detail.html'
-
-
-class ColleagueDetailView(DetailView):
-    model = Colleague
-    template_name = 'colleague_detail.html'
-
-    def get_queryset(self):
-        return Colleague.objects.prefetch_related(
-            Prefetch('placements', queryset=Placement.objects.select_related(
-                'service__assignment'
-            ))
-        )
+        colleague_id = self.request.GET.get('colleague')
+        assignment_id = self.request.GET.get('assignment')
+        
+        if not 'HX-Request' in self.request.headers:
```

Comment:
this threw me off. was wondering how htmx was involved. i think this line can just be removed. this is only called by non-htmx